lib/cli/scripts/init-aps-env.ts

```diff
@@ -435,17 +435,23 @@ async function authorizeUserToContentWithBasic(username: string, contentId: stri
 }
 
 async function downloadLicenseFile(apsLicensePath: string) {
-    try {
-        spawnSync(` aws s3 cp ${apsLicensePath} ./ `, {
-            cwd: path.resolve(__dirname, `./`),
-            shell: false
-        });
-        logger.info(`Aps license file download from S3 bucket`);
-        return true;
-    } catch (error) {
-        logger.error(`Not able to download the APS license from S3 bucket` );
```

Comment:
s3 download was silently failing, but not really sure why

now it's also passing arguments properly via args array
https://nodejs.org/docs/latest-v18.x/api/child_process.html#child_processspawnsynccommand-args-options